### PR TITLE
Glossary related updates

### PR DESCRIPTION
**Pull request by nadinequinr3**
- Clarified usage of the following:
  - Gateway worker
  - P2P Link Manager
  - P2P Link Manager worker
- Updated some term definitions that were long for tooltips
- Added use of "REST user identity" (https://r3-cev.atlassian.net/browse/DOC-5707)
- Changed worker to lowercase
Preview: https://nadine.preview.docs.r3.com/en/platform/corda/5.0.html

### CODE REVIEW

File: content/en/platform/corda/5.0/glossaryitems.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "API": {
         "definition": "APIs allow different software applications to communicate with each other and share data in a standardized and structured way.",
-        "more": "The CorDapp API enables developers using Corda to develop CorDapps by accessing data or functionality from a remote service or application, without needing to understand the underlying implementation details. For more information see the <a href=\"../developing-applications/api.html\"> CorDapp API documentation</a>. "
+        "more": "<ul><li>The CorDapp API enables developers using Corda to develop CorDapps by accessing data or functionality from a remote service or application, without needing to understand the underlying implementation details. For more information see the <a href=\"../developing-applications/api.html\"> CorDapp API documentation</a>.</li><li>The Corda REST API enables you to interact with your CorDapp. For example, to start a flow or retrieve the result of a flow. This API is exposed as a REST service. For more information see the <a href=\"../reference/rest-api.html\"> Corda REST API documentation</a>.</li>"
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.0/glossaryitems.json:4]:
```suggestion
        "more": "<ul><li>The CorDapp API enables developers using Corda to develop CorDapps by accessing data or functionality from a remote service or application, without needing to understand the underlying implementation details. For more information see the <a href=\"../developing-applications/api.html\"> CorDapp API documentation</a>.</li><li>The Corda REST API enables you to interact with your CorDapp; for example, to start a flow or retrieve the result of a flow. This API is exposed as a REST service. For more information see the <a href=\"../reference/rest-api.html\"> Corda REST API documentation</a>.</li>"
```

File: content/en/platform/corda/5.0/glossaryitems.json [hunk 2]
```diff
@@ -92,8 +92,8 @@
...
+        "definition": "A worker designed to communicate with external Corda Clusters via HTTPS. They have restricted access only to specific topics on the Kafka bus and view only the minimum information needed for their role. ",
+        "more": "The gateway workers are the only component of the cluster with access to the wider network and transfer Link Manager session messages taken from Kafka, then using HTTPS POST to send to the peer's gateway worker. After local validation is received, messages are placed onto Kafka for the Link Manager to process. The gateway workers play the important role of establishing and validating the TLS links established to peer Corda Clusters and accessing external resources such as certificate revocation servers, or border proxies. Gateway workers have no direct access to any keys and can only request that the crypto workers sign the TLS handshake data via a specific Kafka topic. Crucially, the gateway worker has no ability to access the keys used in the Ledger, or by the Link Manager."
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.0/glossaryitems.json:96]:
```suggestion
        "more": "The gateway workers are the only component of the cluster with access to the wider network, and transfer Link Manager session messages taken from Kafka, then use HTTPS POST to send them to the peer's gateway worker. After local validation is received, messages are placed onto Kafka for the Link Manager to process. The gateway workers play the important role of establishing and validating the TLS links established to peer Corda Clusters and accessing external resources such as certificate revocation servers, or border proxies. Gateway workers have no direct access to any keys and can only request that the crypto workers sign the TLS handshake data via a specific Kafka topic. Crucially, the gateway worker has no ability to access the keys used in the Ledger, or by the Link Manager."
```

File: content/en/platform/corda/5.0/glossaryitems.json [hunk 3]
```diff
@@ -159,10 +155,14 @@
         "definition": "Public key infrastructure. A comprehensive system of hardware, software, policies, and procedures that enables the secure creation, distribution, management, and revocation of digital certificates and public-key cryptography.",
         "more": ""
     },
-    "P2P Link Manager worker": {
-        "definition": "A Worker that manages the secure transmission of packets to peers. The main functional component is the Link Manager which establishes end-to-end secure sessions for virtual nodes to communicate over.",
+    "P2P Link Manager": {
+        "definition": "Establishes end-to-end secure sessions for virtual nodes to communicate over.",
         "more": "The Link Manager uses the data provided to it by the membership worker to translate virtual node names on CorDapp packets to physical HTTPS destination addresses and requests the gateway worker to transmit its secured packets to the peers. If packets cannot be delivered the Link Manager also manages redelivery, possibly via alternate gateway workers."
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.0/glossaryitems.json:160]:
```suggestion
        "more": "The Link Manager uses the data provided to it by the membership worker to translate virtual node names on CorDapp packets to physical HTTPS destination addresses and requests the gateway worker to transmit its secured packets to the peers. If packets cannot be delivered, the Link Manager also manages redelivery, possibly via alternate gateway workers."
```

File: content/en/platform/corda/5.0/glossaryitems.json [hunk 4]
```diff
@@ -159,10 +155,14 @@
...
     },
+    "P2P Link Manager worker": {
+        "definition": "A worker that manages the secure transmission of packets to peers. The main functional component is the Link Manager which establishes end-to-end secure sessions for virtual nodes to communicate over.",
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.0/glossaryitems.json:163]:
```suggestion
        "definition": "A worker that manages the secure transmission of packets to peers. The main functional component is the Link Manager, which establishes end-to-end secure sessions for virtual nodes to communicate over.",
```

File: content/en/platform/corda/5.0/glossaryitems.json [hunk 5]
```diff
@@ -172,12 +172,12 @@
         "more": ""
     },
     "REST worker": {
-        "definition": "A Worker that exposes the intranet HTTPS REST control ports of the Corda cluster. The REST workers support all the dynamic administration APIs of the cluster, such as APIs to configure the cluster, or setup user role based permissions, or create and register virtual nodes. They also provide the virtual node endpoint APIs for invoking the flows and the ledger applications and retrieving their status/results.",
-        "more": "All REST calls must be authenticated and authorized, typically using OAUTH2 for login/authentication and Role Based Access Controls checked against a user permissions database."
+        "definition": "A worker that exposes the intranet HTTPS REST control ports of the Corda cluster. The REST workers support all of the dynamic administration APIs of the cluster, such as APIs to configure the cluster, or setup user role based permissions, or create and register virtual nodes. ",
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.0/glossaryitems.json:175]:
```suggestion
        "definition": "A worker that exposes the intranet HTTPS REST control ports of the Corda cluster. The REST workers support all of the dynamic administration APIs of the cluster, such as APIs to configure the cluster, set up user role based permissions, or create and register virtual nodes. ",
```

File: content/en/platform/corda/5.0/glossaryitems.json [hunk 6]
```diff
@@ -172,12 +172,12 @@
...
+        "more": "REST workers also provide the virtual node endpoint APIs for invoking the flows and the ledger applications and retrieving their status/results. All REST calls must be authenticated and authorized, typically using OAUTH2 for login/authentication and Role Based Access Controls checked against a user permissions database."
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.0/glossaryitems.json:176]:
```suggestion
        "more": "REST workers also provide the virtual node endpoint APIs for invoking the flows and the ledger applications, and for retrieving their status/results. All REST calls must be authenticated and authorized, typically using OAUTH2 for login/authentication and Role Based Access Controls checked against a user permissions database."
```

File: content/en/platform/corda/5.0/glossaryitems.json [hunk 7]
```diff
@@ -172,12 +172,12 @@
...
     },
     "Revocation check": {
-        "definition": "In the context of X.509 certificates, revocation checks refer to the process of verifying whether a digital certificate is still valid and has not been revoked by the issuing Certificate Authority (CA). Revocation checks are typically performed using mechanisms such as Certificate Revocation Lists (CRLs) or the Online Certificate Status Protocol (OCSP), which provide up-to-date information on the revocation status of certificates.",
-        "more": ""
+        "definition": "In the context of X.509 certificates, revocation checks refer to the process of verifying whether a digital certificate is still valid and has not been revoked by the issuing Certificate Authority (CA). ",
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.0/glossaryitems.json:179]:
```suggestion
        "definition": "In the context of X.509 certificates, revocation checks refer to the process of verifying if a digital certificate is still valid and has not been revoked by the issuing Certificate Authority (CA). ",
```

**Review by paulmoloneyr3 — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/5.0/glossaryitems.json (hunk 2 above)
Comment (nadinequinr3) [reply content/en/platform/corda/5.0/glossaryitems.json:96]: What about? 
```suggestion
        "more": "The gateway workers are the only component of the cluster with access to the wider network. The workers transfer Link Manager session messages taken from Kafka and use HTTPS POST to send them to the peer's gateway worker. After local validation is received, messages are placed onto Kafka for the Link Manager to process. The gateway workers play the important role of establishing and validating the TLS links established to peer Corda Clusters and accessing external resources such as certificate revocation servers, or border proxies. Gateway workers have no direct access to any keys and can only request that the crypto workers sign the TLS handshake data via a specific Kafka topic. Crucially, the gateway worker has no ability to access the keys used in the Ledger, or by the Link Manager."
```

Comment (paulmoloneyr3) [reply content/en/platform/corda/5.0/glossaryitems.json:96]: Yep that works!

**Review by yarivus — APPROVED** [review]
LGTM

**Review by paulmoloneyr3 — APPROVED** [review]

**Pull request merged**